components/gif_picker/components/Search/index.jsx

```diff
@@ -5,9 +5,12 @@ import React, {PureComponent} from 'react';
 import PropTypes from 'prop-types';
 import {connect} from 'react-redux';
 
+import {throttle} from 'lodash';
+
 import {searchIfNeededInitial, searchGfycat} from 'mattermost-redux/actions/gifs';
 
 import SearchGrid from 'components/gif_picker/components/SearchGrid';
+import {CUSTOM_EMOJI_GIF_SEARCH_THROTTLE_TIME_MS} from '../../../emoji_picker/emoji_picker';
```

Comment:
Could you actually define a separate constant for these two? While they have the same value, I think it's better to reduce the coupling between these components by just duplicating the constant

Reply:
Done. Seperate constants are defined